src/locale/phrases/phrases-nb.js

```diff
@@ -112,27 +112,27 @@ const phrases = {
         PRIVATE: 'Kladd',
         PUBLISHED: 'Offentlig',
         UNLISTED: 'Privat',
-        SUBMITTED: 'Til offentliggjøring',
```

Comment:
Dette stemmer ikkje heilt. Statusen er heller noko sånt som 'Be om publisering'.

Reply:
Oppdatert til "Be NDLA om publisering"